.circleci/config.yml

```diff
@@ -76,6 +78,12 @@ jobs:
       - run: date +%F > date
       - restore_cache:
           <<: *restore_cache
+      - run:
```

Comment:
Why are there no changes in the frontend tests job? If we're replacing codecov with code climate, then it should be throughout the code base.

Reply:
I don't think the front end test job is even working with codecov. I debugged this by doing cat ../karma_coverage_reports/coverage-final.json and its just printing an empty dict ({}). So we can't do this with code-climate, i.e we cannot upload an empty json object to the code-climate server

Reply:
We have an issue filed for that (#6821). We'll be resolving it soon. If that's the case, please add necessary commands here as well with a TODO to uncomment them when #6821 is resolved.

Reply:
done